manuscript/270_Mock_Objects.md

```diff
@@ -118,7 +118,7 @@ Received.InOrder(() =>
 };
 ```
 
-What it actually does is checking whether the `Destination` got the messages (remember? Objects send messages to each other) in the right order. If we changed the implementation of the `ApplyTo()` method from this one:
+What it does is checking whether the `Destination` got the messages (remember? Objects send messages to each other) in the right order. If we changed the implementation of the `ApplyTo()` method from this one:
```

Comment:
:+1: (by the way, this is still a very draft content, so it will probably change considerably)